draft-packages/well/docs/Well.stickersheet.stories.tsx

```diff
@@ -0,0 +1,102 @@
+import React from "react"
+import { Meta, StoryFn } from "@storybook/react"
+import { StickerSheet } from "../../../storybook/components/StickerSheet"
+import { Well } from "../index"
+
+export default {
+  title: "Stickersheets/Well",
+} satisfies Meta<typeof Well>
+
+const ExampleContent = (): JSX.Element => <>&nbsp;</>
```

Comment:
Maybe this shouldn't be empty?

Reply:
Wanted to simplify the snapshots by avoiding importing any new components, but perhaps a sentence will do